libs/wingsdk/src/cloud/secret.ts

```diff
@@ -53,14 +58,17 @@ export abstract class Secret extends Resource {
 
   /** @internal */
   public _getInflightOps(): string[] {
-    return [SecretInflightMethods.VALUE, SecretInflightMethods.VALUE_JSON];
+    return [
+      SecretInflightMethods.GET_VALUE,
+      SecretInflightMethods.GET_VALUE_JSON,
+    ];
   }
 }
 
 /**
  * Options when getting a secret value
  */
-export interface GetSecretValueOptions {
+export interface ISecretGetValueOptions {
```

Comment:
```suggestion
export interface SecretGetValueOptions {
```

This is a jsii struct, not an interface.